EOFTests/eof_validation/stack/exchange_stack_validation.json

```diff
@@ -0,0 +1,201 @@
+{
+  "exchange_stack_validation": {
+    "vectors": {
+      "exchange_stack_validation_0": {
+        "code": "0xef00010100040200010017040000000080000a6001600160016001600160016001600160016001e80000",
+        "results": {
+          "Prague": {
+            "result": true
+          }
+        }
+      },
+      "exchange_stack_validation_1": {
+        "code": "0xef00010100040200010017040000000080000a6001600160016001600160016001600160016001e81000",
+        "results": {
+          "Prague": {
+            "result": true
+          }
+        }
+      },
+      "exchange_stack_validation_10": {
+        "code": "0xef00010100040200010017040000000080000a6001600160016001600160016001600160016001e81600",
+        "results": {
+          "Prague": {
+            "result": true
+          }
+        }
+      },
+      "exchange_stack_validation_11": {
+        "code": "0xef00010100040200010017040000000080000a6001600160016001600160016001600160016001e86100",
+        "results": {
+          "Prague": {
+            "result": true
+          }
+        }
+      },
+      "exchange_stack_validation_12": {
```

Comment:
Besu has exchange_stack_validation_12 through exchange_stack_validation_18 as all valid.  For each case there are 10 items on the stach, and the excanges are (9, 9), (1, 9), (8, 9) (2, 9), (5, 9), (6, 9), and (4, 9).  All work with 10 stack items.

Reply:
The correct action may be to change the spec of EXCHANGE, so it swaps `n+2` and `n+m+3`, where 1 is the top of the stack.  So `e900` would swap the second and third item, and would also result in no combos such as (9,9) which are pointless.  This would prevent any operation from swapping with the top of the stack, which SWAPN covers, as well as swapping the same two items.

With that re-definition the pairs are (10, 11), (2, 11), (9, 11), (3, 11), (6, 11), (7, 11), and (5, 11), all of which bust the stack and make these tests valid.

Reply:
Ok, that every term has a +1 is where my brain took shortcuts and presumed eveything had an off by one error.  they should be folded into one final use.